docs/input/docs/reference/configuration.md

```diff
@@ -52,16 +53,22 @@ commit-date-format: yyyy-MM-dd
 merge-message-formats: {}
 update-build-number: true
 semantic-version-format: Strict
-strategies: [ConfigNext, MergeMessage, TaggedCommit, TrackReleaseBranches, VersionInBranchName]
+strategies:
+- ConfiguredNextVersion
+- MergeMessage
+- TaggedCommit
+- TrackReleaseBranches
+- VersionInBranchName
 branches:
   develop:
-    mode: ContinuousDeployment
+    take-incremented-version: TakeAlwaysIncrementedVersion
```

Comment:
Hm, I think `take-incremented-version: TakeAlwaysIncrementedVersion` have room for improvements. Give me some time to think through the use case and see if I can come up with better names.

Reply:
Let's do as discussed on Slack and group this and `prevent-increment-of-merged-branch-version` into a new top-level property as such:

```yaml
prevent-increment:
    when-versioned-branch-merged: true
    when-tagged: true
```